### DISCUSSION TITLE
Release 0.2.2

### DISCUSSION THREAD

**Discussion by evpobr** [libsndfile/libsamplerate#168] [General]
@SoapGentoo , what do you think? We have some changes enough for minor release.

**Comment by SoapGentoo** [comment]
sure why not, on the weekend then?

**Reply by evpobr** [reply]
Ok.

**Reply by evpobr** [reply]
@SoapGentoo , release when you ready, i will add Windows ZIPs.

**Reply by SoapGentoo** [reply]
everything ready from my side, LTMK when the ZIPs are ready

**Comment by evpobr** [comment]
[libsamplerate-0.2.2-win32.zip](https://github.com/libsndfile/libsamplerate/files/7111557/libsamplerate-0.2.2-win32.zip)
[libsamplerate-0.2.2-win64.zip](https://github.com/libsndfile/libsamplerate/files/7111559/libsamplerate-0.2.2-win64.zip)

**Comment by SoapGentoo** [comment]
uploaded with sigs

**Comment by evpobr** [comment]
Thanks 👍

**Comment by evpobr** [comment]
[Vcpkg](https://github.com/microsoft/vcpkg) port update PR:

microsoft/vcpkg#20003
